homeassistant/components/reolink/entity.py

```diff
@@ -50,3 +47,32 @@ def __init__(
     def available(self) -> bool:
         """Return True if entity is available."""
         return self._host.api.session_active and super().available
+
+
+class ReolinkNVRCoordinatorEntity(CoordinatorEntity):
```

Comment:
Where is this used?

Reply:
This is not yet used, I removed it for now.
I will add this back in some form once the first NVR entities are beeing added in a next PR.